keyvault/client/src/main/java/com/azure/keyvault/implementation/RestPagedResponseImpl.java

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.keyvault.implementation;
+
+import com.azure.common.http.HttpHeaders;
+import com.azure.common.http.HttpRequest;
+import com.azure.common.http.rest.RestPagedResponse;
+
+import java.util.List;
+
+/**
+ * Response of a REST API that returns page.
+ *
+ * @param <T> the type items in the page
+ */
+public class RestPagedResponseImpl<T> implements RestPagedResponse<T> {
```

Comment:
Same here @conniey

Reply:
Noted, needs to be moved to Azure common.